web/src/modules/user/profile/reducer.ts

```diff
@@ -101,6 +102,12 @@ const passwordChangeReducer = (state: ProfileState['passwordChange'], action: Pr
                 ...state,
                 error: action.error,
             };
+        case PROFILE_CHANGE_PASSWORD_RESET:
+            return {
+                ...state,
+                success: false,
+            };
+
```

Comment:
why do you need separate action for this? Success state of reset password is being changed to false on `PROFILE_CHANGE_PASSWORD_FETCH`

Reply:
really i can use it for it ```PROFILE_CHANGE_PASSWORD_FETCH``` but still it will difficult to read.
We need it because the component updated when the user types his password (we send calls to the barong(password-change-meter)) and in this case component trigger componentWillReceiveProps, and in this moment next.passwordChangeSuccess will equal to true because the last action for it was ```PROFILE_CHANGE_PASSWORD_DATA```.

Reply:
ok clear, thank you